Sprint-1/exercises/decimal.js

```diff
@@ -1,4 +1,7 @@
 const num = 56.5678;
+let wholeNumberPart=Math.floor(num);
+let decimalPart=num-wholeNumberPart;
```

Comment:
Because of round off error in floating point number arithmetic, when you output the value of `decimalPart`. console.log() would output the its value with more than 4 decimal places. This is normal. 

Can you modify the code so that only 4 decimal places of `decimalPart` is outputted?